.github/workflows/cargo.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.55.0
```

Comment:
Why was this required? The main repo use 1.56.0, it would be good to align on that if possible.

Reply:
it seems to work. 
here is the ci running with that version: https://github.com/jurgob/examples/pull/1